custom_components/opnsense/coordinator.py

```
@@ -119,144 +106,146 @@
+                    server_name
+                    not in dict_get(self._state, "telemetry.openvpn.servers", {}).keys()
+                ):
+                    continue
 
-            lease_stats: Mapping[str, int] = {"total": 0, "online": 0, "offline": 0}
-            for lease in self._state["dhcp_leases"]:
-                if "act" in lease.keys() and lease["act"] == "expired":
+                if (
+                    server_name
+                    not in dict_get(
+                        self._state, "previous_state.telemetry.openvpn.servers", {}
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)